pkg/admission/interfaces.go

```diff
@@ -64,6 +64,7 @@ type Operation string
 const (
 	Create  Operation = "CREATE"
 	Update  Operation = "UPDATE"
+	Patch   Operation = "PATCH"
```

Comment:
Patch and update are the same thing semantically, why do they need separate entries here?